communication/mavlink_waypoint_handler.cpp

```diff
@@ -878,14 +878,17 @@ void Mavlink_waypoint_handler::auto_landing_handler()
         {
             case Navigation::DESCENT_TO_SMALL_ALTITUDE:
                 print_util_dbg_print("Cust: descent to small alt");
-                state_.mav_mode_custom &= static_cast<mav_mode_custom_t>(0xFFFFFFE0);
-                state_.mav_mode_custom |= CUST_DESCENT_TO_SMALL_ALTITUDE;
+                state_.mav_mode_custom &= static_cast<Mav_mode::custom_mode_t>(0xFFFFFFE0);
+                state_.mav_mode_custom |= Mav_mode::CUST_DESCENT_TO_SMALL_ALTITUDE;
+                waypoint_hold_coordinates.waypoint = position_estimation_.local_position;
```

Comment:
Nop, I think this is now done in the fonction that set the autolanding, so that you can either land on spot or at a given location.
this refer to setting waypoint_hold_coordinates